src/VirtualDeviceValidator.ts

```diff
@@ -67,6 +67,7 @@ export class VirtualDeviceValidator {
             currentSequenceIndex += 1;
             if (currentSequenceIndex === 1) {
                 await this.virtualDevice.message("Alexa, exit");
+                await this.virtualDevice.resetSession();
```

Comment:
Isn't this redundant with the "alexa exit" - or do we believe exit works better than quit for clearing the session?

I also mean for this reset to be done on the first call to the virtual device (so, if the first call is anything other than resetSession, then it is made to resetSession).

Reply:
- We clear the session by using the reserved keyword `quit`.
- We believe is the right keyword to end the session:

> "Exit" and "Quit" are reserved words for a 'hard exit' on a Session. 

ref: https://github.com/alexa/skill-sample-nodejs-fact/issues/3#issuecomment-343783805

> Quit is implemented by Amazon. You can't do anything about it. I think some people have reported problems when you give a reply to the End Session event. Check to make sure you make no reply to End Session. 

ref: https://forums.developer.amazon.com/questions/5938/error-with-utterances-quit-exit-remote-endpoint.html

- Let's remove the redundancy and keep only `resetSession`, addressed on: https://github.com/bespoken/virtual-device-sdk/pull/50/commits/4b735ee055c5e3075482cea9d5381413096219a5#diff-6ff04c4b228419c3a6e0c79dd43df867R69